src/main/java/edu/gatech/cs2340/trydent/GameObject.java

```diff
@@ -25,7 +25,7 @@
 
     private String name = "Untitled GameObject";
 
-    private Group fxNode;
+    protected Group fxNode;
```

Comment:
I'll make it private with a getter.

I can't really think of a use case where someone would want to directly re-assign the node. Though that doesn't mean one doesn't exist. But people can construct GameObjects with arbitrary Node objects and have them added to the scene graph in a predictable way, so I feel like that covers everything.